minetest.conf.example

```diff
@@ -1630,7 +1630,7 @@
 #    Name of map generator to be used when creating a new world.
 #    Creating a world in the main menu will override this.
 #    Current stable mapgens:
-#    v5, v6, v7 (except floatlands), flat, singlenode.
+#    v5, v6, v7 (except floatlands), singlenode.
```

Comment:
where did `flat` go?

Reply:
See end of first post in #8251 , de-stabilised for dungeon reasons, but at least v5 is now stable to make the 'old 3' stable.